README.md

```diff
@@ -54,7 +54,7 @@ via the standard commands, i.e.,
 
 In addition, algorithms not denoted with "\*" above are not enabled for
 TLS operations. This designation can be changed by modifying the
-"enabled" flags in the main [algorithm configuration file](oqs-template/generate.yml)
+"enabled" flags in the main [algorithm configuration file](oqs-template/generate-extras.yml)
```

Comment:
This change is not correct; my bad to not have done #109 earlier: When that lands, the original reference is correct, so please revert this change.